src/components/Omr/OmrItem.vue

```diff
@@ -240,7 +283,7 @@ export default defineComponent({
        * - question is non-graded and the given option has been selected
        * @param {Number} optionIndex - index of the option
        */
-    function optionBackgroundClass(optionIndex: number) {
+    function optionBackgroundClass(answer: number | string) {
```

Comment:
No need of mentioning that it could be array here?

Reply:
no because answer here is for a particular option. an option is either interpeted as `number` (for single choice, multi choice --> it would be option index) or `string` (for matrix matching)